BackofficeBundle/Resources/config/role_parameter.yml

```diff
@@ -0,0 +1,116 @@
+parameters:
+    open_orchestra_backoffice.role:
+        role_access_tree_general_node :
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_general_node }
```

Comment:
you can use this syntax : 

        role_access_tree_general_node :
          category: open_orchestra_backoffice.left_menu.editorial.title
          label: open_orchestra_role.label_access_general_node

i think this syntax is more explicit